conf.d/gitnow_config.fish

```diff
@@ -8,11 +8,19 @@ set -q fish_functions; or set -g fish_functions "$fish_config/functions"
 set -q fish_completions; or set -g fish_completions "$fish_config/completions"
 set -q GITNOW_CONFIG_FILE; or set -g GITNOW_CONFIG_FILE ~/.gitnow
 
-set -g gitnow_version 2.6.0
+set -g gitnow_version 2.7.0
```

Comment:
Why that change? Are you using the latest master? Can you sync your PR branch please?
https://github.com/joseluisq/gitnow/blob/master/conf.d/gitnow_config.fish#L11

Reply:
Oh my bad. I simply pulled from master when forking, then made the `fundle-compat` branch after editting the file.

Reply:
Wait, on master it's 2.6 too.
I honestly don't know what happened there, that is quite odd. I'll revert it.

Reply:
Basically you need to sync your fork first.